# Case Study 01: Sales Force Training

### DataFrame

Perusahaan X ingin meningkatkan penjualan mereka. Dari data penjualan sebelumnya menunjukkan bahwa penjualan rata-rata yaitu $100 per transaksi. Setelah melakukan training kepada pekerja sales, data penjualan terbaru (yang diambil dari 25 sampel pekerja sales) tersimpan dalam tabel di bawah ini:

In [6]:
import numpy as np
import pandas as pd
from scipy import stats
import statistics

In [7]:
TransactionAmount = [100,150,50,100,130,120,100,85,70,150,150,120,50,100,100,140,90,150,50,90,120,100,110,75,65]
type(TransactionAmount)

list

### Measure of Central Tendency

In [8]:
# Rata-Rata
print("Rata-Rata: ",statistics.mean(TransactionAmount))

# Median
print("Nilai Tengah: ",statistics.median(TransactionAmount))

# Modus
print("Nilai yang Paling Sering Muncul: ",statistics.mode(TransactionAmount))

Rata-Rata:  102.6
Nilai Tengah:  100
Nilai yang Paling Sering Muncul:  100


### Measure of Variability

In [9]:
# Standard Deviasi
print("Standart Deviasi: ", np.std(TransactionAmount))

# Variansi
print("\nVariansi: ",np.var(TransactionAmount))

# Range
print("\nNilai Minimal: ",np.min(TransactionAmount))
print("Nilai Maksimal: ",np.max(TransactionAmount))
print("Range: ",np.max(TransactionAmount) - np.min(TransactionAmount))

#Quartil
print("\nNilai Q1 (0,25): ", np.percentile(TransactionAmount,25))
print("Nilai Q2 (0,5): ", np.percentile(TransactionAmount,50))
print("Nilai Q3 (0,75: )", np.percentile(TransactionAmount,100))

Standart Deviasi:  31.18076330047101

Variansi:  972.2399999999999

Nilai Minimal:  50
Nilai Maksimal:  150
Range:  100

Nilai Q1 (0,25):  85.0
Nilai Q2 (0,5):  100.0
Nilai Q3 (0,75: ) 150.0


### Hypothesis Testing

#### Mendefinisikan Hipotesis

###### H0 -> Transaksi rata-rata per hari = 100
###### H1 -> Transaksi rata-rata per hari > 100

###### _Alpha = 0.05_

In [10]:
len(TransactionAmount)

25

In [11]:
# T-statistic & p-value

print("Hasil: ",stats.ttest_1samp(a=TransactionAmount,popmean=100))

Hasil:  TtestResult(statistic=np.float64(0.4085001556802841), pvalue=np.float64(0.6865284813438117), df=np.int64(24))


##### Kesimpulan korelasi alpha dengan p-value

1. Testing menolak hipotesis H0
3. Kampanye tidak efektif karena transaksi rata-rata masih pada level 100

In [12]:
# T-statistic & critical region

print("Hasil Critical Region: t > ",stats.t.ppf(1-0.05,24))

Hasil Critical Region: t >  1.7108820799094275


#### Kesimpulan Akhir

1. Hasil T-statistic = 0.4
2. Hasil Critical Region = t>1.71 

3. Testing menolak hipotesis H0 karena hasil critical region lebih besar daripada hasil t-statistic
4. Kampanye tidak efektif karena transaksi rata-rata tetap pada level 100